v1.1-events.yml

```diff
@@ -262,8 +262,10 @@ definitions:
     properties:
       type:
         type: string
+        x-validation: true
       id:
         type: string
+        x-validation: true
```

Comment:
Seems like we lie and say some other fields are also validated when they clearly aren't!